wfe/wfe.go

```diff
@@ -1955,6 +1995,22 @@ func uniqueLowerNames(names []string) []string {
 	return unique
 }
 
+// uniqueIPs returns the set of all unique IP addresses in the input.
+// The returned names will be sorted in assending order.
+// this internally call UniqueLowerNames as map doesn't support net.IP as key
```

Comment:
I think this function would be clearer/faster if you built a map using the `ip.String()` as the key and didn't rely on `uniqueLowerNames`. That will also avoid having to re-parse the IP back from the string.

E.g. something like:
```
uniqMap := make(map[string]net.IP)
for _, ip := range IPs {
  uniqMap[ip.String()] = ip
}
results := make([]net.IP, len(uniqMap))
for k, v := range uniqMap {
  results = append(result, v)
}
return results
```